packages/cfd/src/Components/cfd-personal-details-form.tsx

```diff
@@ -126,6 +129,7 @@ export const InputField = ({ maxLength, name, optional = false, ...props }: TCFD
 
 const validatePersonalDetails = ({
     values,
+    context,
```

Comment:
Why are we passing down the `RootStore` like this?

Reply:
because we need to pass the `store` from appstore to the cfd for those Components which are exported to be used out of the cfd package